src/Application/Redux/reducers.js

```diff
@@ -70,7 +70,15 @@ function trumpMM(state = StateUtils.getInitialState(), action) {
       return Object.assign({}, state, { news: state.news.addArticle(bounceArticle) });
 
     case UPGRADE_DENOMINATION:
-      return Object.assign({}, state, { mint: state.mint.toNextDenomination });
+      let mintDelta;
+      let newsArticle;
+      if (state.bank.cash >= state.mint.nextDenomination.priceToUnlock) {
+        mintDelta = { mint: state.mint.toNextDenomination };
+        newsArticle = `Trump's greatness has improved the value of US currency!`;
+      } else {
+        newsArticle = `Trump's experiments in alchemy have failed to produce results`;
```

Comment:
no doubt, more thought is needed for how to best mix in the humor with useful stuff.  in lieu of that thought, i figured continuing on the same path can't hurt for now.  also the wording was off the cuff, we can think more about that too (it doesn't include information about which upgrade level you're at now, for instance)